### PR TITLE
feat: Removed check and unittests from az pipeline

### PR DESCRIPTION
**Pull request by Mossaka**
I've noticed that, in avg, it takes 17 mins to execute `make improve` on windows machines. Considering that we are running both `make improve` and unittests on GH action, there is no need to run them again in our integration pipeline. This PR removes them, and hopefully will make our integration pipeline faster to finish. 

Signed-off-by: Jiaxiao Zhou <jiazho@microsoft.com>

### CODE REVIEW

File: build/azure-pipeline/workflow-pr.yml [hunk 1]
```diff
@@ -172,11 +156,8 @@ stages:
           #   displayName: Build C Examples
           
           #
-          # Running all tests
+          # Running integration tests
           #
-          - script: make test
```
Comment (danbugs) [inline build/azure-pipeline/workflow-pr.yml:177]: some tests might use environment variables, I don't think we should remove tests (e.g., #153)

**Review by danbugs — CHANGES_REQUESTED** [review]
~one small request, otherwise, great idea!

File: build/azure-pipeline/workflow-pr.yml (hunk 1 above)
Comment (Mossaka) [reply build/azure-pipeline/workflow-pr.yml:177]: we have github action dedicated to running unittests, so I believe it's okay to not run them again in our integration pipelines.

**Review by danbugs — APPROVED** [review]
LGTM

**Pull request merged**